From 1e68f53c89c3e0f71fdb409b404dff538208bd59 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Dalleau?= <frederic.dalleau@docker.com>
Date: Mon, 14 Mar 2022 12:39:00 +0100
Subject: [PATCH] Disable default_permission in virtio_fs.c

---
 fs/fuse/virtio_fs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/fuse/virtio_fs.c b/fs/fuse/virtio_fs.c
index b9cfb1165ff4..a2d43c24c4c5 100644
--- a/fs/fuse/virtio_fs.c
+++ b/fs/fuse/virtio_fs.c
@@ -1282,7 +1282,7 @@ static const struct fuse_iqueue_ops virtio_fs_fiq_ops = {
 static inline void virtio_fs_ctx_set_defaults(struct fuse_fs_context *ctx)
 {
 	ctx->rootmode = S_IFDIR;
-	ctx->default_permissions = 1;
+	ctx->default_permissions = 0;
 	ctx->allow_other = 1;
 	ctx->max_read = UINT_MAX;
 	ctx->blksize = 512;
-- 
2.32.0

